src/library/scala/collection/mutable/PriorityQueue.scala

```diff
@@ -254,12 +320,24 @@ sealed class PriorityQueue[A](implicit val ord: Ordering[A])
    *
    *  @return  a priority queue with the same elements.
    */
-  override def clone(): PriorityQueue[A] = new PriorityQueue[A] ++= this.iterator
+  override def clone(): PriorityQueue[A] = {
+    val pq = new PriorityQueue[A]
+    for (i <- 1 to this.length) pq.unsafeAdd(resarr(i))
+    pq
```

Comment:
Maybe you want to mention, if you leave it this way, that you don't need to call `heapify` because the heap is already in order!  (Otherwise it's a bit jarring: "wait, didn't you say never to return after unsafeAdd without calling heapify?")